WiserTaskScheduler/WiserTaskScheduler/Modules/Branches/Services/BranchQueueService.cs

```diff
@@ -530,17 +530,27 @@ FROM INFORMATION_SCHEMA.ROUTINES
 
                 // Rollback transaction if started
                 await databaseConnection.RollbackTransactionAsync(false);
+                logger.LogError(exception, $"Failed to create the branch '{settings.DatabaseName}'.");
```

Comment:
Please use the logService for logging. This ensures the log is written to the database and/or a Slack channel.

Reply:
Done